rebound/horizons.py

```diff
@@ -111,6 +111,20 @@ def getParticle(particle=None, m=None, x=None, y=None, z=None, vx=None, vy=None,
 
         made_choice = True
         body = api_request(idn, datestart, dateend, plane)
+    elif "Matching small-bodies" in body:
+        for line in body.split("\n"):
+            try:
+                first_word = line.split()[0]
+            except IndexError:
+                continue
+            print(first_word)
```

Comment:
@hannorein I'm sorry, it seems like a print() statement has slipped into this PR. I will create a PR to remove it once I get around to it.

Reply:
And I think this block also needs a `made_choice = True`.

Reply:
I agree..